examples/vision/xray_classification_with_tpus.py

```diff
@@ -0,0 +1,453 @@
+"""
+Title: Pneumonia Classification on TPU
+Author: Amy MiHyun Jang
+Date created: 2020/07/28
+Last modified: 2020/08/05
+Description: Medical image classification on TPU
+"""
+"""
+## Introduction + Set-up
+
+This tutorial will explain how to build an X-ray image classification model
+to predict whether an X-ray scan shows presence of pneumonia.
+"""
+
+import re
+import os
+import random
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+try:
+    tpu = tf.distribute.cluster_resolver.TPUClusterResolver()
+    print("Device:", tpu.master())
+    tf.config.experimental_connect_to_cluster(tpu)
+    tf.tpu.experimental.initialize_tpu_system(tpu)
+    strategy = tf.distribute.experimental.TPUStrategy(tpu)
+except:
+    strategy = tf.distribute.get_strategy()
+print("Number of replicas:", strategy.num_replicas_in_sync)
+
+"""
+We need a Google Cloud link to our data to load the data using a TPU.
```

Comment:
Please add a couple lines about the pre-requisites to run on TPU: must be on Colab with the TPU runtime selected.